src/TestData.php

```diff
@@ -75,12 +75,11 @@ private function humanizeClass($name)
         return ucfirst($name);
     }
 
-    public function update(\Closure $closure)
+    public function removeAbsolutePath($absolutePath)
     {
-        $closure->call($this);
+        $this->file = ltrim(preg_replace('~^'. preg_quote($absolutePath, '~') . '~', '', $this->file), DIRECTORY_SEPARATOR);
```

Comment:
probably better implementation:

```suggestion
        if (str_starts_with($this->file, $absolutePath)) {
            $this->file = ltrim(substr($this->file, strlen(absolutePath)), DIRECTORY_SEPARATOR);
        }
```